src/libs/OptionsListUtils.js

```diff
@@ -372,6 +373,40 @@ function getAllReportErrors(report, reportActions) {
     return allReportErrors;
 }
 
+/**
+ * Get the preview message to be displayed in the option list.
+ *
+ * @param {Object} report
+ * @param {Object} reportAction
+ * @param {Boolean} [isPreviewMessageForParentChatReport]
+ * @returns {String}
+ */
+function getReportPreviewMessageForOptionList(report, reportAction, isPreviewMessageForParentChatReport = false) {
```

Comment:
@luacmartins I think we can just set default value here:
```suggestion
function getReportPreviewMessageForOptionList(report, reportAction = {}, isPreviewMessageForParentChatReport = false) {
```
This was old code: https://github.com/Expensify/App/pull/31061/files#diff-65c096044d5f69b35bcdec14c99c4fda4580759df9b1a7c36650d58eea276f1dR1861

Reply:
that will prevent the crash, yes, but why don't we have any reportActions for the given report? When I load the given report I do see reportActions

Reply:
it may be that the reportActions are only missing for a brief moment while loading/updating onyx